GIT 提交訊息：

huan5678 opened the pull request:
功能：增強專案設置與使用者體驗改進

- 為 Vue 3 新增 ESLint 配置並覆寫設定
- 將 `simple-import-sort` 插件整合到 ESLint 規則中
- 更改 `package.json` 中的專案名稱
- 引入新的 npm 腳本以進行 linting 和 Husky 設定
- 為預提交掛鉤添加 `lint-staged` 配置
- 為特色和服務新增新的 SVG 圖標
- 包含新增的 `about.json`、`privacy.json`、`terms.json` 數據檔案
- 介紹帶有對應路由的 `AboutView`、`PrivacyView`、`TermsView` 以及 `NotFoundView` Vue 組件
- 更新 Tailwind CSS 配置，加入新的顏色和列表樣式類型
- 在 Vite 配置中啟用 ESLint 插件
- 將所有無法匹配的路由重定向至 404 頁面
- 移除 `main.js` 中不必要的導入語句

huan5678 committed:
GIT 提交訊息：

功能：增強專案設置與使用者體驗改進

- 為 Vue 3 新增 ESLint 配置並覆寫設定
- 將 `simple-import-sort` 插件整合到 ESLint 規則中
- 更改 `package.json` 中的專案名稱
- 引入新的 npm 腳本以進行 linting 和 Husky 設定
- 為預提交掛鉤添加 `lint-staged` 配置
- 為特色和服務新增新的 SVG 圖標
- 包含新增的 `about.json`、`privacy.json`、`terms.json` 數據檔案
- 介紹帶有對應路由的 `AboutView`、`PrivacyView`、`TermsView` 以及 `NotFoundView` Vue 組件
- 更新 Tailwind CSS 配置，加入新的顏色和列表樣式類型
- 在 Vite 配置中啟用 ESLint 插件
- 將所有無法匹配的路由重定向至 404 頁面
- 移除 `main.js` 中不必要的導入語句

Signed-off-by: 黃士桓 <huan5678@gmail.com>

eee317 commented on src/views/PrivacyView.vue:
```diff
@@ -0,0 +1,50 @@
+    <section class="flex flex-col gap-8">
+      <ol class="flex flex-col gap-6">
+        <li v-for="rule in rules" :key="rule.title" class="space-y-6">
+          <h2 class="text-xl font-bold leading-[1.4]">{{ rule.title }}</h2>
```
leading-[1.4] 要不要直接在tailwind.config.js 檔案內的 extend裡面新增一個專門 1.4 的呀？因為我在後面還有看到幾個地方有 leading-[1.4] 的寫法
```js
extend: {
      lineHeight: {
        'extra-loose': '1.4',
      }
```

huan5678 committed:
GIT提交訊息：

樣式：「重構用戶介面樣式以保持一致性」

- 為保持一致性，在各Vue組件中更新類別排序並移除行高
- 通過移除清單樣式和更新螢幕斷點來簡化TailwindCSS配置
- 在多個視圖中標準化領先（行高）

Signed-off-by: 黃士桓 <huan5678@gmail.com>

huan5678 committed:
[update] color patten
